introduction.md

```diff
@@ -1,2 +1,10 @@
-Hello World!
+#Hello World!
 My favourite Code School path is Ruby!
+
+## Deadly Skills
+* Git
+* JavaScript
+* Ruby
+* C/C++/C#
+* Java
+* PHP
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?